src/components/Widget/components/Conversation/components/Header/index.js

```diff
@@ -1,17 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import close from 'assets/clear-button.svg';
 import './style.scss';
 
-const Header = ({ title, subtitle }) =>
+const Header = ({ title, subtitle, toggleChat, showCloseButton }) =>
   <div className="header">
+    {showCloseButton &&
```

Comment:
leave the `{` in a separate line

Reply:
Mmm.. I don't really like the way it looks writing in a separate line and leaving the `{` alone :disappointed: